src/njoy21/input/HEATR/Card1/Nplot.hpp

```diff
@@ -14,11 +14,13 @@ struct Nplot {
       "ascii text files. Negative values indicate unformatted input i.e.\n"
       "Fortran-style block binary\n"
       "\n"
-      "nplot values are restricted to an absolute value between 20 and 99,\n"
-      "inclusively, and are required to\n"
-      " + have an absolute value distinct from the corresponding nendf value\n"
-      " + have an absolute value distinct from the corresponding nin value\n"
-      " + have an absolute value distinct from the corresponding nout value";
+      "If the nplot option is used, then the nploy value are restricted to\n"
```

Comment:
`If the nplot option is used, then the plot values are restricted to an absolute value`...

You should write here that `nplot` != 0 should only be used when `iprint`=2 from Card2